lib/iglu-client/resolver.rb

```diff
@@ -16,11 +16,12 @@ module Iglu
 
   # Iglu Client. Able to fetch schemas only from Iglu Central
   class Resolver
-    attr_reader :registries, :cache
+    attr_reader :registries, :cache, :cache_ttl
```

Comment:
I expect to see test coverage for this new feature with **at least** following cases:

* Parse resolver config JSON with `cacheTtl`
* Invalidate schema after some period of time (make sure new fetch attempt is undertaken). You can use `sleep` here
* Confirm that no new fetch attempts are undertaken after small enough period